livekit-agents/livekit/agents/voice_assistant/assistant.py

```diff
@@ -327,7 +332,52 @@ async def _main_task(self) -> None:
         )
 
         self._ready_future.set_result(None)
-        await self._update_co()
+
+        # Loop running each 10ms to do the following:
+        #  - Update the volume based on the user speech probability
+        #  - Decide when to interrupt the agent speech
+        #  - Decide when to validate the user speech (starting the agent answer)
+        speech_prob_avg = utils.MovingAverage(100)
+        speaking_avg_validation = utils.MovingAverage(150)
+        interruption_speaking_avg = utils.MovingAverage(
+            int(self._opts.int_speech_duration * 100)
+        )
+
+        interval_10ms = aio.interval(0.01)
+
+        vad_pw = 2.4  # TODO(theomonnom): should this be exposed?
```

Comment:
what does this value indicate?

Reply:
the amount of volume you decrease/increase based on the speech probability

Reply:
it's more like "how fast" you increase/decrease